index.html

```diff
@@ -76,20 +92,22 @@ <h2>Tonic</h2>
         </div>
       </div>
       <div class="works_card_action">
-        <button class="secondary">See Project</button>
+        <button class="project-btn">See Project</button>
+        </div>
       </div>
     </div>
   </div>
```

Comment:
[OPTIONAL]
- [ ] For a better cleaner code, kindly remove the extra closing `div` tag on line `99`.